Tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+"""Configuration file for the test suite."""
+
+import os
+import sys
+
+GO_BACK_A_DIRECTORY = "/../"
+GO_INTO_SRC_DIRECTORY = "src"
+
+# set the system path to contain the previous directory
+PREVIOUS_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, PREVIOUS_DIRECTORY + GO_BACK_A_DIRECTORY + GO_INTO_SRC_DIRECTORY)
```

Comment:
This code here technically does not do anything since there is no `src` directory currently